Source/FieldSolver/ElectrostaticSolver.cpp

```diff
@@ -218,6 +221,7 @@ WarpX::AddSpaceChargeFieldLabFrame ()
     bool const do_rz_volume_scaling = false;
     for (int ispecies=0; ispecies<mypc->nSpecies(); ispecies++){
         WarpXParticleContainer& species = mypc->GetParticleContainer(ispecies);
+        if (species.do_not_deposit) continue;
```

Comment:
This is a picky detail, but perhaps use the same construction as above, checking for `(!species.do_not_deposit)`. This would make it easier to compare the two routines if needed in the future.

Reply:
In general, I prefer to maintain shallower nesting, but given that it's just one statement, I agree with you here. Done.